src/components/Form/Form.js

```diff
@@ -134,37 +134,43 @@ export function Form({ children, config }) {
       })
     }
 
+    const fieldComponent = {
```

Comment:
I had to refactor the field hook to only add labelCopy if the fieldConfig passed it in--CheckboxInput does not use this and I do not see why it should have to since it's children prop will have verbiage to make intent clear.

Otherwise, we'll  call back the children hook with `labelCopy` and React will error that this is not a known property for the CheckboxInput component.